src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/ReviewEvidenceNote.cshtml

```diff
@@ -33,20 +31,46 @@
         @Html.Partial("_ViewEvidenceNotePartial", Model.ViewEvidenceNoteViewModel)
 
         <!-- radiobuttons for approved -->
-        <p>Choose whether to approve, reject or return the evidence note</p>
+        <p class="govuk-!-margin-top-7">Choose whether to approve, reject or return the evidence note</p>
         <div class="govuk-form-group @Html.Gds().FormGroupClass(m => m.SelectedValue)" id="approval-options">
             @Html.Gds().ValidationMessageFor(m => m.SelectedValue)
             @this.WeeeGds().RadioButtonsFor(
                 m => m.SelectedValue,
                 Model.PossibleValues,
-                "Choose whether to approve, reject or return the evidence note",
-                RadioButtonLegend.VisuallyHidden,
+                " ",
+                RadioButtonLegend.Displayed,
                 RadioButtonLayout.Stacked,
                 Model.HintItems)
         </div>
 
+        <div id="reason-text" style="display: none;" class="govuk-!-margin-bottom-7">
+            <div class="govuk-body">What is the reason you are rejecting or returning the evidence note?</div>
+            @Html.TextAreaFor(m => m.Reason, htmlAttributes: new { @class = "govuk-input", @id = "return-reject-reason", @name = "return-reject-reason", @style = "height: 100px;", @maxlength = "200", @title = "You can type up to 200 characters" })
+            <!-- textarea data-bind="*Model.Reason" class="govuk-input" id="return-reject-reason" name="return-reject-reason" style="height: 100px;" maxlength="200" title="You can type up to 200 characters"></textarea -->
```

Comment:
You should be able to use govuk-textarea style. Have a look at https://design-system.service.gov.uk/components/textarea/ rather than setting a fixed height in px.